src/indexer/utils.js

```diff
@@ -47,6 +47,9 @@ function parseLocatorAndLocatorType(bytes32Locator, identifier, protocol) {
     locator = identifier.toLowerCase()
   } else if (locatorType === 'https') {
     locator = ethers.utils.parseBytes32String(bytes32Locator)
+    if (!_.startsWith(locator, 'https://')) {
+      locator = `https://${locator}`
+    }
```

Comment:
This wouldn't work if someone added a non SSL locator. We could use `!locator.includes('http')` to account for that case, but only if we want to support those who aren't technically adhering to the protocol by using a non SSL locator

Reply:
I thought about that, but http should be a different protocol type than https (and hence a different index). This index type should be strictly https only.

Reply:
If someone wanted to use websockets that would be another protocol type, and server sent events would be another protocol type. Trying to "fix" the protocol type to account for mistakes breaks the model, I'd rather have it fail outright.